packages/protocol/contracts/L1/gov/TkoGv.sol

```diff
@@ -14,21 +14,17 @@
 
 pragma solidity 0.8.24;
 
-import "@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol";
 import
     "@openzeppelin/contracts-upgradeable/governance/compatibility/GovernorCompatibilityBravoUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesUpgradeable.sol";
 import
     "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesQuorumFractionUpgradeable.sol";
 import
     "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorTimelockControlUpgradeable.sol";
 import "../../common/OwnerUUPSUpgradable.sol";
 
-contract TaikoGovernor is
+contract TkoGv is
```

Comment:
Hmm... not sure we should reduce the size by renaming the contract...

Reply:
How other people did it.

Reply:
Yeah, but please help me out how can we, bc. we are now on the edge.

Reply:
> How other people did it.

I see multiple folks running into these type of issues - if we aggregating to many voting schemes (Compound style bravo upgrade support, etc.)
e.g.:
https://forum.openzeppelin.com/t/if-i-use-the-optimizer-to-optimize-the-code-is-there-any-problem-in-using-all-the-functions-provided-by-openzeppelins-governor/30298
https://forum.openzeppelin.com/t/governor-contract-code-size-is-32070-bytes-and-cannot-be-deployed/36408/8

With optimizer disbaled, we can rename it back.. tho.. Will look into it a bit more.

Reply:
Was able to rename it back to TaikoGovernor on the price of having "Taiko" in `__Governor_init` for the domain separator, i think that is a fair compromise (for now), thanks Brecht for the extra pair of eyes.